dockerbuild/all-in-one/README.md

```diff
@@ -0,0 +1,38 @@
+This Dockerfile and its sibling scripts are from an earlier configuration of ModelDB. They are preserved here but not required by other Dockerfiles.
```

Comment:
Maybe rename this folder to archive? b/c the other docker compose stuff is also all-in-one in some sense.